prefetch.bs

```diff
@@ -523,6 +576,31 @@ The <dfn>list of sufficiently strict speculative navigation referrer policies</d
           <div class="note">This remove-and-insert pattern is consistent with what happens when [=receiving a cookie=].</div>
 </div>
 
+<div algorithm>
+  <div class="note">
+    This is essentially how the <a http-header lt="Cookie">`` `Cookie` ``</a> header is computed, except that a few details are specialized to this case, the last-access-time is not modified, and the sort order is changed to ensure it is unique.
+  </div>
+
+  To <dfn>gather cookie data</dfn> for [=URL=] |url| in cookie store |cookieStore|:
+
+  1. Let |cookieList| be the set of cookies from |cookieStore| that meet all of the following requirements:
+
+      * one of the following applies:
```

Comment:
I am nervous about keeping this in sync with the cookies spec, especially given all the proposals that are floating around. But I don't have any good ideas.

Reply:
Yeah, that was one reason for the note explicitly pointing out where it came from. This duplication isn't great but it wasn't obvious to me either how to avoid it.